apps/examples/tls_selftest/Makefile

```diff
@@ -66,6 +66,8 @@ ASRCS =
 CSRCS =
 MAINSRC = tls_selftest_main.c
 
+CFLAGS += -I$(TOPDIR)/../external/include/
```

Comment:
Why do you add this? `external/include` is a basic folder of including. You don't need to do it again.

Reply:
Without this, a build error occurs since "mbedtls/config.h" is not found...
Could you know how to resolve that kind of error..?

Reply:
use `<mbedtls/config.h>` for basic including header which is located in os/include, external/include, framework/include

Reply:
Well, I don't know why but <> also doesn't work.. In fact, I tried it already..

Reply:
What Make.defs did you use in build/configs? Could you check ARCHINCLUDES?

Reply:
This is definitely unnecessary.

Reply:
I found that to update ARCHINCLUDES works.
Thanks. I'll modify.